messaging-activemq/subsystem/src/main/java/org/wildfly/extension/messaging/activemq/AddressSettingAdd.java

```diff
@@ -192,13 +161,11 @@ static AddressSettings createDefaultAddressSettings() {
         settings.setAutoCreateAddresses(AddressSettingDefinition.AUTO_CREATE_ADDRESSES.getDefaultValue().asBoolean());
         settings.setAutoDeleteAddresses(AddressSettingDefinition.AUTO_DELETE_ADDRESSES.getDefaultValue().asBoolean());
         settings.setAutoDeleteCreatedQueues(AddressSettingDefinition.AUTO_DELETE_CREATED_QUEUES.getDefaultValue().asBoolean());
-        settings.setMaxReadPageBytes(AddressSettingDefinition.MAX_READ_PAGE_BYTES.getDefaultValue().asInt());
```

Comment:
Why is that removed ??

Reply:
@ehsavoie For the same reason as the other one, in this case the default for `maxReadPageBytes` is `2 * pageSizeBytes`

Reply:
It has been set to this value for backward compatible reasons due to changes in paging

Reply:
It's the same issue. We can end up in a situation where we're ignoring user setting because our default for a MAX_VALUE is less than a user-defined BASE_VALUE.